schema/deploy/mutations/custom_create_naics_mutation.sql

```diff
@@ -0,0 +1,20 @@
+-- Deploy ggircs-portal:mutations/custom_create_naics_mutation to pg
+-- requires: tables/naics
+
+begin;
+
+create or replace function ggircs_portal.custom_create_naics_mutation(
+  naics_code_input int,
```

Comment:
A naics code is a varchar made of numbers, not an int.
I has to follow a lexicographic order, e.g. 11 is closer to 1111 than to 21.
In general, all of these "code" fields should be a varchar, because you never know when someone is going to decide that a new `11-a`  code will be created